src/browser-polyfill.js

```diff
@@ -491,19 +494,7 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
       get: {minArgs: 1, maxArgs: 1},
       set: {minArgs: 1, maxArgs: 1},
     };
-    apiMetadata.privacy = {
-      network: {
-        networkPredictionEnabled: settingMetadata,
-        webRTCIPHandlingPolicy: settingMetadata,
-      },
-      services: {
-        passwordSavingEnabled: settingMetadata,
-      },
-      websites: {
-        hyperlinkAuditingEnabled: settingMetadata,
-        referrersEnabled: settingMetadata,
-      },
-    };
+    apiMetadata.privacy = {"*": {"*": settingMetadata}};
```

Comment:
I'm not immediately against this implementation, since it is still somewhat readable and concise.
I do wonder whether we should explicitly spell out `network`, `services`, and `websites`.

@rpl Thoughts?

Reply:
@Rob--W Any thought on this? Happy to explicitly spell out `network`, `services`, and `websites`, if that holds the progress.

CC: @rpl

Reply:
:+1: from me on spelling out the first level and use the "*" to cover all the properties inside those.

Reply:
Thanks @rpl and @Rob--W, done.

I was thinking about 2 different implementation:
```
const privacyType = {"*": settingMetadata};
apiMetadata.privacy = {
      network: privacyType,
      services: privacyType,
      websites: privacyType,
};
```

```
apiMetadata.privacy = {};
for (const type of ["network", "services", "websites"]) {
  apiMetadata.privacy[type] = {"*": settingMetadata};
}
```

If you have preference for one or another please let me know which to use. This seem to be most consistent comparing to `settingMetadata`.

Reply:
Keep the current code. It is more readable than the proposed alternatives.